server/chat-plugins/help.ts

```diff
@@ -0,0 +1,473 @@
+/**
+ * PS Help room auto-response plugin.
+ * Uses Regex to match room frequently asked question (RFAQ) entries,
+ * and replies if a match is found.
+ * Supports configuration.
+ * Written by mia-pi.
+ */
+
+import {FS} from '../../lib/fs';
+import {Utils} from '../../lib/utils';
+import {LogViewer} from './chatlog';
+import {ROOMFAQ_FILE} from './room-faqs';
+const PATH = 'config/chat-plugins/help.json';
+const MINIMUM_LENGTH = 7;
+const helpRoom = Rooms.get('help');
```

Comment:
this constant not being CONSTANT_CASE while the others are is odd to me. This should also be configurable?

Reply:
I mean it's fairly easy to configure, you just change the Rooms.get(); 
I can make it an actual config setting but that seems.. annoying.

Reply:
I don't consider it a constant. It's certainly not immutable.

Reply:
Like I could see:

```ts
const HELP_ROOMID = 'help';
const helpRoom = Rooms.get(HELP_ROOMID);
```